test/e2e/checkpoint_test.go

```diff
@@ -334,6 +334,10 @@ var _ = Describe("Podman checkpoint", func() {
 		IPBefore.WaitWithDefaultTimeout()
 		Expect(IPBefore.ExitCode()).To(Equal(0))
 
+		MACBefore := podmanTest.Podman([]string{"inspect", "-l", "--format={{.NetworkSettings.MacAddress}}"})
```

Comment:
I don't know if we actually expect these to be the same, at present. `podman checkpoint` seems to have special code for restoring IP address, which I don't think is duplicated for MAC yet.

I don't know if we care, at this point - it might be safe to remove this test, which is the only thing not passing?

Reply:
Well, I am afraid that my changes pass the requested MAC to container networks but there is nothing reading it and changing network interface.

Can you give me a hint where I need to add the missing MAC handler, please?

Reply:
The changes you made to pod network get it into OCICNI, so I think this should work from our end. You might want to add some debug to make absolutely sure the settings are getting to `getPodNetwork()`. It they are, it's probably somewhere in `ocicni`.